### PR TITLE
Add BDN9 Macropad

### PR DESCRIPTION
**Pull request by nooges**
## Description
Adds BDN9 Macropad

## Types of changes

- [ ] Core
- [ ] Bugfix
- [ ] New Feature
- [ ] Enhancement/Optimization
- [x] Keyboard (addition or update)
- [x] Keymap/Layout/Userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by this PR

* 

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document. (https://docs.qmk.fm/#/contributing)
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mechmerlin** [comment]
Any interest in making a keebio directory to put this in?

File: keyboards/bdn9/bdn9.h [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+/* This a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+#define LAYOUT( \
```
Comment (drashna) [inline keyboards/bdn9/bdn9.h]: What, no ortho_3x3?!  /s

:D

**Comment by drashna** [comment]
Awesome, thanks!

**Comment by drashna** [comment]
And I agree, a keebio folder may be a good idea, since there are a lot of keebio keyboards now:

* Iris
* Levinson
* Nyquist
* Viterbi
* BFO
* Laplace
* BDN9
* Quefrency
* Fourier
* Chocopad
* Big Switch? (not sure about this one)

Wow, yeah, there are a good number here!

**Comment by nooges** [comment]
Yeah, I think I'll get those moved in and sync up all of my USB VID/PIDs as well.

Big Switch wasn't designed by me, and it does have an official VID/PID: http://pid.codes/1209/B195/

**Comment by mechmerlin** [comment]
Wow, that's a lot! Putting it in a vendor directory also helps discoverability of boards made by a certain vendor! 

I didn't know you made the chocopad/BFO!

**Comment by drashna** [comment]
I wasn't sure about the big switch. I was just looking at your store for stuff listed there.  

:)

**Pull request merged**